recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,25 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    recipe_arr = list(recipe.values())
+    ingredient_arr = list(ingredients.values())
+    how_much = []
+    if len(recipe_arr) > len(ingredient_arr):
+        return 0
+    if len(recipe_arr) == 1:
+        return math.floor(ingredient_arr[0] / recipe_arr[0])
+    #  Above if statement is because the for loop wouldn't even start if range was (0, 0) just a minor bug
+    #  I'll fix it if I get the chance
+    for i in range(0, len(recipe_arr)-1):
```

Comment:
I think you can actually get rid of the above if statement if you change what you're iterating over.
Try out 
`for ingredient, amount in recipe.items():`
remember you have access to items because recipe and ingredients are dicts
that worked for me.